routes/web.php

```diff
@@ -13,7 +13,8 @@
     'prefix' => config('cookieconsent.url.prefix'),
 ], function() {
     Route::get('script', ScriptController::class)
-        ->name('script');
+        ->name('script')
+        ->middleware('cache.headers:public;max_age=2628000;etag');
```

Comment:
Could you please explain why this line is necessary and how it is related to the issue you're solving? Thanks!